language-design.md

```diff
@@ -79,4 +101,7 @@ you do so? If not, why not?
 
 **Response**
 
+We both agreed that the two experiences mentioned are not necessarily at odds with one another. Both the researchers' comments and Cook's quotes reflect the challenge of creating a programming language that resembles natural language. Cook mentioned that even though the language appears to be a natural language, it is actually an artificial language that can be confusing for users to write, due to the special cases and idiosyncrasies of the language. Both experiences highlight the difficulties in designing a programming language that effectively balances ease of use and readability with accuracy and consistency. When designing a DSL, we had agreements and disagreements. Carson said that he may include little natural language so that the DSL is understood and used by a larger audience, even beginners, rather than only for people who are experienced in some programming languages. A way that it may be done is, for example, using the natural language when using loops. It may be confusing for someone who doesn't know Java to see a for loop in Java and instantly understand what is going. Using a natural language here can make it much easier for newer people.
```

Comment:
When answering this question I took a stance that was more in line with Salim's reasoning, having a natural language in a DSL would make writing code confusing. However, I thought that Carson made a really good point that I didn't consider in my response, that having a natural language would make it easier for non-coders to write programs. Ultimately, I agree with you both, a including a natural language in a DSL can prove to be a powerful and helpful tool.